oc-chef-pedant/lib/pedant/rspec/search_util.rb

```diff
@@ -323,6 +323,25 @@ def can_perform_basic_searches_for(object_type)
           end
         end # can_perform_basic_searches_for
 
+        # Search for limited rows to see if we get the correct number back
```

Comment:
Similarly even though it's implicit, we should verify that a search that returns less than the expected number of results will return no results for the next page.